packages/components/src/composite/stories/index.story.tsx

```diff
@@ -93,116 +92,112 @@ const meta: Meta< typeof UseCompositeStorePlaceholder > = {
 };
 export default meta;
 
-export const Default: StoryFn< typeof UseCompositeStorePlaceholder > = (
-	storeProps
-) => {
-	const rtl = isRTL();
-	const store = useCompositeStore( { rtl, ...storeProps } );
-
-	return (
-		<Composite store={ store }>
-			<Composite.Item>Item one</Composite.Item>
-			<Composite.Item>Item two</Composite.Item>
-			<Composite.Item>Item three</Composite.Item>
-		</Composite>
-	);
+export const Default: StoryObj< typeof Composite > = {
+	args: {
+		rtl: isRTL(),
```

Comment:
Mmm, I'm having some thoughts about this `rtl` feature.

One, I see we're still referring to `store` in the description here so we should rewrite that.

https://github.com/WordPress/gutenberg/blob/955ac5de18decdb5eaec7aa394b9771437ced14f/packages/components/src/composite/types.ts#L117-L127

Two, `rtl: isRTL()` as part of the args object is not going to work dynamically with the Storybook's RTL switcher, unlike when it was executed as part of the story function.

Three, this then raises the question of whether we should have `rtl={ isRTL() }` be the default behavior for the component. I can imagine a lot of consumers forgetting to do this, and they can override it if they need to. What do you think? At the very least I think this practice should be featured more heavily in the docs.

Reply:
> we're still referring to `store` in the description here so we should rewrite that.

👍 

> `rtl: isRTL()` as part of the args object is not going to work dynamically with the Storybook's RTL switcher

Good point — it may work if Storybook does a full re-render, but it any case it's better to improve it on our end

> whether we should have `rtl={ isRTL() }` be the default behavior for the component.

I agree that this would be a healthy default.

Reply:
https://github.com/WordPress/gutenberg/pull/64723/commits/12f7d1063623cab9f7f1b7382c461735f1049d28